îº§ pytest testrunner.py; python3 main.py
============================= test session starts ==============================
platform linux -- Python 3.8.10, pytest-6.2.1, py-1.10.0, pluggy-0.13.1
rootdir: /home/runner/TestModule-Task1-1
collected 17 items                                                             

testrunner.py ..........F......                                          [100%]

=================================== FAILURES ===================================
_______________________________ test_check_psw8 ________________________________

    def test_check_psw8():
      psw = "Qwert5@"
>     assert check_psw(psw) is False
E     AssertionError: assert True is False
E      +  where True = check_psw('Qwert5@')

testrunner.py:67: AssertionError
----------------------------- Captured stdout call -----------------------------
Valid password
=========================== short test summary info ============================
FAILED testrunner.py::test_check_psw8 - AssertionError: assert True is False
========================= 1 failed, 16 passed in 0.19s =========================


